packages/dev/core/src/Collisions/gpuPicker.ts

```diff
@@ -160,6 +195,21 @@ export class GPUPicker {
             this._readbuffer = new Uint8Array(engine.isWebGPU ? 256 : 4); // Because of block alignment in WebGPU
         }
 
+        // Do we need to rebuild the RTT?
+        const size = this._pickingTexure!.getSize();
+
+        if (size.width !== rttSizeW || size.height !== rttSizeH) {
+            this._pickingTexure!.dispose();

```

Comment:
You could do it inside `_createRenderTarget` to save one or two code line.

Reply:
yes!